awslabs/mxnet-model-server
# Remove unused python integration_tests.

1. Remove integration_tests
2. Update test README.md
3. remove unused error code.

*Issue #, if available:*

*Description of changes:*


By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
